frontend/styles/Timeline.module.tsx

```diff
@@ -0,0 +1,144 @@
+import styled from '@emotion/styled';
+import { styled as mui_styled } from '@mui/material/styles';
+import { Tabs, Tab } from '@mui/material';
+
+/* #313638, #feb14b, #e85f5c, #f3f3f3 */
```

Comment:
We'll likely need a separate PR where these colours are put in the root so we can access them via some key words rather than hex codes since they're used very frequently, but don't worry about it for this one.

Reply:
Noted and removed.